README.md

```diff
@@ -1,2 +1,45 @@
-# new-relic
-Best practice New Relic configurations.
+# New Relic considerations
+
+Considerations for using New Relic in the federal government.
+
+Below is a description of the settings in your New Relic `.ini` file that concern privacy and security. Example configuration files can be found in this repository:
```

Comment:
It's not an `.ini` file for other languages...maybe just say "your New Relic config file" and link to [the docs](https://docs.newrelic.com/docs/apm/new-relic-apm/installation-configuration/configure-agent#agent-config)?